test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1

```diff
@@ -1,45 +1,137 @@
+            $driveProp.PSDrive.Name | Should Be $psDriveName
+        }
+
+        It "Remove the PSDrive" {
+            $existsBefore = Test-Path "${psDriveName}:\"
+            Remove-PSDrive -Name ${psDriveName} -ea SilentlyContinue
+            $existsAfter = Test-Path "${psDriveName}:\"
+            $existsBefore | Should Be $true
+            $existsAfter | Should Be $false
+        }
+
+        It "Verify 'Used' and 'Free' script properties" {
+            $drive = Get-PSDrive -Name $psDriveName
+            $drive.Used -gt 0 | Should Be $true
+            $drive.Free -gt 0 | Should Be $true
+        }
+    }
+}
+
+Describe "Extended Alias Provider Tests" -Tags "Feature" {
     BeforeAll {
         #just use same location as TestDrive for simplicity
         $psDriveRoot = "TestDrive:"
         $psDriveName = "PsTestDriveName"
     }
 
-    BeforeEach {
-        New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot > $null
-    }
+    Context "Valdiate New-PSDrive Cmdlet Parameters" {
+        AfterEach {
+            Remove-PSDrive -Name $psDriveName -ErrorAction SilentlyContinue
+        }
 
-    AfterEach {
-        Remove-PSDrive -Name $psDriveName -ErrorAction SilentlyContinue
-    }
+        It "Verify Description" {
+            $result = New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot -Description "Test PSDrive to remove"
+            $result.Description | Should Be "Test PSDrive to remove"
+        }
 
-    It "Create a new PSDrive" {
-        $newDrive = New-PSDrive -Name "NewDifferentPSDrive" -PSProvider FileSystem -Root $psDriveRoot
-        try {
-            $newDrive.Name | Should Be "NewDifferentPSDrive"
-            $newDrive.Root | Should Be (Convert-Path $psDriveRoot)
+        It "Verify Confirm" {
+            $result = New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot -Confirm:$false
+            $result.Name | Should Be $psDriveName
         }
-        finally {
-            Remove-PSDrive -Name "NewDifferentPSDrive" -ErrorAction SilentlyContinue
+
+        It "Verify WhatIf" {
+            New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot -WhatIf > $null
+            try {
+                Get-PSDrive -Name $psDriveName -ErrorAction Stop
+                throw "Expected exception not thrown"
+            }
+            catch { $_.FullyQualifiedErrorId | Should Be "GetLocationNoMatchingDrive,Microsoft.PowerShell.Commands.GetPSDriveCommand" }
         }
-    }
 
-    It "Read data from a PSDrive" {
-        $driveProp = Get-ItemProperty ${psDriveName}:
-        $driveProp.PSDrive.Name | Should Be $psDriveName
+        It "Verify Scope" {
+            New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot -Description "Test PSDrive to remove" -Scope Local > $null
+            $foundGlobal = $true
+            try { $globalDrive = Get-PSDrive -Name $psDriveName -Scope Global -ErrorAction Stop }
```

Comment:
Do you mean use multiple lines for the try or because line 81 is a particularly long line?